### PR TITLE
Remove old worker code.

### PR DESCRIPTION
**Pull request by klopyrev**
@percyliang

### CODE REVIEW

File: codalab/objects/dashboard.ws [hunk 1]
```diff
@@ -29,8 +29,7 @@ These are bundles that are currently running or queued to be run.
 % add time time duration
 % add state
 % display table r
-% search state=running created=.sort- .limit=10000 .mine
-% search state=queued created=.sort- .limit=10000 .mine
+% search state=staged,making,waiting_for_worker_startup,starting,running created=.sort- .limit=10000 .mine
```
Comment (percyliang) [inline codalab/objects/dashboard.ws]: Might as well add 'created' here too

**Comment by percyliang** [comment]
LGTM

circleci is failing - check it out

File: codalab/objects/dashboard.ws (hunk 1 above)
Comment (klopyrev) [inline codalab/objects/dashboard.ws]: Done.

**Comment by klopyrev** [comment]
CircleCI tests pass now.

**Comment by klopyrev** [comment]
Subsumed by https://github.com/codalab/codalab-cli/pull/482

**Pull request closed without merging**